doc/howtos/booting-and-installation.rst

```diff
@@ -147,6 +147,126 @@ Flashing an Intel Edison requires use of a breakout board and two micro-USB cabl
 #. Plug in the second micro-USB cable to the J16 connector as instructed by the running flashall script.
 #. Wait for all the images to flash. You will see the progress on the flasher.
 #. Once flashing is done, the image will automatically boot up and auto-login as ``root``, no password is required.
+   
+BeagleBone Black
+================
+
+BeagleBone Black is booted from a microSD card with MBR (Master Boot Record) and not GPT (GUID Partition Table) partitions.
+Most freshly unpackaged microSD cards come with MBR partitions, but previously used ones might not.
+
+You'll probably need an adapter to use the microSD card on your host computer. If you use a microSD-to-SD adapter, 
+it will likely show up as /dev/mmcblk0. You can verify the device name assigned by using ``dmesg`` or the
+``lsblk`` command to look for the device name for the microSD card (check for a device with the size you're expecting). 
+
+In our examples below, we're using an 8GB microSD card in an SD adapter that's showing up as /dev/mmcblk0  
+(numbers and device name maybe different for your device and system).
+
+You should download the ``ostro-image-dev-beaglebone*.rootfs.tar.bz2`` image from the `Ostro Project download server`_
+(if you're not doing your own build), and from that same folder, you'll also need the device tree 
+blob file (``zImage-am335x-boneblack.dtb```) 
+used by the kernel to describe the BeagleBone Black hardware.
+
+.. _Ostro Project download server: http://download.ostroproject.org
+
+.. comment:  next steps derived from http://www.armhf.com/boards/beaglebone-black/bbb-sd-install/
+
+#. First, verify the microSD card is using MBR partitions. Run ::
```

Comment:
Shouldn't we recommend to first unmount the partitions (in case they got automatically mounted), I got a 'Re-reading the partition table failed.: Device or resource busy' initially when I went through these steps.